src/test/java/de/retest/recheck/ui/diff/ElementDifferenceFinderTest.java

```diff
@@ -0,0 +1,49 @@
+package de.retest.recheck.ui.diff;
+
+import static de.retest.recheck.ui.diff.ElementDifferenceFinder.getNonEmptyDifferences;
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Test;
+
+import de.retest.recheck.ui.descriptors.Element;
+
+public class ElementDifferenceFinderTest {
+
+	@Test
```

Comment:
New tests should use JUnit 5, then we can also omit `public` both on class and method level.

Reply:
Yes, but this is a copy&paste test from retest...